app/Db/thermocon.db

```diff
@@ -22,6 +22,25 @@ record(calc, "$(DEV):TEMP_RBV")
     field(EGU,  "C")
 }
 
+record(ai, "$(DEV):EXT_TEMP_RAW_RBV")
+{       
+#    field(PINI, "YES")
+    field(DESC, "Raw external sensor temp")
+    field(DTYP, "asynInt32")
+    field(INP,  "@asyn(THERMOCON$(N)_read_reg,1)UINT16")
+    field(SCAN, "1 second")
+    field(VAL,  "0")
+    field(FLNK, "$(DEV):EXT_TEMP_RBV")
+}
+ 
+record(calc, "$(DEV):EXT_TEMP_RBV")
+{
+    field(DESC, "external sensor temp in C")
+    field(INPA, "$(DEV):EXT_TEMP_RAW_RBV PP NMS")
+    field(CALC, "A&32768?-1*((~A) + 1)/100:A/100")
```

Comment:
A note that this is converting 2's complement 16-bit negative numbers may be worthwhile (with the ternary operator that's supported by the [calc record](https://epics.anl.gov/base/R7-0/7-docs/calcRecord.html) - `?`)

Reply:
@klauer Good idea. I added this above. LMK what you think.

Reply:
LGTM 👍